tests/unit/via/get_url_details_test.py

```diff
@@ -14,18 +14,34 @@
 
 
 class TestGetURLDetails:
+    GOOGLE_API_KEY = "google_api_key"
+
     @pytest.mark.parametrize(
         "content_type,status_code", (("text/html", 501), ("application/pdf", 200))
     )
     def test_it_calls_get_for_normal_urls(self, requests, content_type, status_code):
-        requests.get.return_value = self._make_response(content_type, status_code)
+        response = Response()
+        response.raw = BytesIO(b"")
+        response.headers = {"Content-Type": content_type}
+        response.status_code = status_code
+        requests.get.return_value = response
```

Comment:
This was based on a branch which re-used this code. We only use it once now so I've in-lined it.